service/src/MSRIOGroup.cpp

```diff
@@ -195,13 +195,23 @@ namespace geopm
         register_signal_alias("CPU_POWER_LIMIT_CONTROL", "MSR::PKG_POWER_LIMIT:PL1_POWER_LIMIT");
         register_signal_alias("CPU_POWER_TIME_WINDOW", "MSR::PKG_POWER_LIMIT:PL1_TIME_WINDOW");
 
+        // CNLIOGroup provides a CNL::BOARD_ENERGY & BOARD_ENERGY high level alias.
+        // If it loads after the MSRIOGroup the BOARD_ENERGY alias below will be
+        // overwritten, so both MSR::BOARD_ENERGY & BOARD_ENERGY are provided
+        register_signal_alias("MSR::BOARD_ENERGY", "MSR::PLATFORM_ENERGY_STATUS:ENERGY");
+        register_signal_alias("BOARD_ENERGY", "MSR::BOARD_ENERGY");
```

Comment:
```suggestion
        register_signal_alias("BOARD_ENERGY", "MSR::PLATFORM_ENERGY_STATUS:ENERGY");
```
I don't see any other occurrences of an alias of an alias.  One less complexity to deal with when calling into this IOGroup.